src/SmartComponents/Vulnerability/VulnerabilityCard.js

```diff
@@ -122,18 +120,15 @@ class VulnerabilityCard extends Component {
                         linkDescription="Last 30 days"
                     />
                     <VerticalDivider/>
-                    <NumberDescription
-                        data="NA"
-                        dataSize="md"
-                        linkDescription="Last 7 days"
-                    />
+                    { latestVulnerabilitiesFetchStatus === 'fulfilled' && (
+                        <NumberDescription
+                            data={ latestVulnerabilities.meta.total_items }
+                            dataSize="md"
+                            linkDescription="Last 7 days"
```

Comment:
Need to build this string out in messages.js, gotta keep up with translations 🦹‍♀️

tho.. abstracting the string, making it something yah gotta pass the number of days into AND using the pluralizing `days` would make this a really handy string (rather than only translating `last 7 days`)

Reply:
In regards to the `last 7 days` I was going to keep working on this card to pull in data from the APIs, but instead im going to leave that up to the dev working on the card, and then i can always come back to it to clean it up.